charts/osdfir-infrastructure/charts/openrelik/templates/gcp/frontend-backendconfig.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if (and (.Values.global.ingress.enabled) (eq .Values.global.ingress.className "gce")) }}
+apiVersion: cloud.google.com/v1
+kind: BackendConfig
+metadata:
+  name: {{ .Release.Name }}-openrelik-frontend-backend-config
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  timeoutSec: 300
+  healthCheck:
+    checkIntervalSec: 300
+    timeoutSec: 5
+    healthyThreshold: 2
+    unhealthyThreshold: 2
+    type: HTTP
```

Comment:
you want to explicitly specify the requestPath as / ?

Reply:
It doesn't hurt to to be clear (its the default otherwise), I updated to add requestPath: /